packages/sui-widget-embedder/package.json

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
Where's the README ?
Really necesary at least to know how to set up a project with widgets.

Reply:
Yep, that is he first commit, I am working on it.